ddk-target/ddk.target

```diff
@@ -39,5 +32,14 @@
     <repository location="http://download.eclipse.org/lsp4j/updates/releases/0.10.0"/>
     <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.10.0.v20201105-1605"/>
   </location>
+  <location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
+    <repository location="https://download.eclipse.org/tools/orbit/downloads/drops/R20200529191137/repository"/>
+    <unit id="org.hamcrest.core" version="1.3.0.v20180420-1519"/>
+    <unit id="org.hamcrest.core.source" version="1.3.0.v20180420-1519"/>
+    <unit id="org.hamcrest.library" version="1.3.0.v20180524-2246"/>
+    <unit id="org.hamcrest.library.source" version="1.3.0.v20180524-2246"/>
+    <unit id="org.mockito" version="2.23.0.v20200310-1642"/>
+    <unit id="org.mockito.source" version="2.23.0.v20200310-1642"/>
+  </location>
 </locations>
-</target>
+</target>
```

Comment:
Can we put back the removed end of line?

Reply:
done